stable_diffusion_videos/stable_diffusion_walk.py

```diff
@@ -170,7 +170,7 @@ def walk(
         data = json.load(open(prompt_config_path))
         prompts = data['prompts']
         seeds = data['seeds']
-        num_steps = data['num_steps']
+        num_interpolation_steps = data['num_interpolation_steps']
```

Comment:
Maybe we should have some backwards compatibility here - check first if `num_interpolation_steps` exists in prompt config. If not, check if `num_steps` is there.

Reply:
how can i do that ?